### ISSUE TITLE
Game window larger than the screen on first run on Windows

### ISSUE THREAD

**Issue by illwieckz** [DaemonEngine/Daemon#708]
`Sir_Toodles` on Discord reported this experience on Windows:

```
<Sir_Toodles> yo how come when i open the game its quite zoomed in and i cant access anything
<illwieckz> is it the first time you try the game?
<Sir_Toodles> yes
<illwieckz> hmm, is the game window larger than your screen?
<Sir_Toodles> yes
<illwieckz> are you running linux, windows or macos?
<Sir_Toodles> windows
```

I asked him to send me his `autogen.cfg` file, which had this content:

```
// generated by Unvanquished, do not modify
seta name Studmuffin
seta r_customwidth 1920
seta audio.al.device ""
seta cg_welcome 1
seta r_customheight 1080
seta language en_AU
```

I sent him this one and asked him to replace his `autogen.cfg` file with this one and restart the game :

```
// generated by Unvanquished, do not modify
seta name Studmuffin
seta audio.al.device ""
seta cg_welcome 1
seta r_customwidth 640
seta r_customheight 480
seta r_mode -1
seta r_fullscreen off
seta language en_AU
```

This fixed the issue for him.

FYI those options are options to force the game to start as a 640×480 window to make sure the game isn't larger than the screen, the user has to resize to his preferences using the in game options after that:

```
seta r_customwidth 640
seta r_customheight 480
seta r_mode -1
seta r_fullscreen off
```

**Comment by slipher** [comment]
```
<perturbed> @Sir_Toodles how many monitors do you have and what are their resolutions?
<Sweet> and did you manage to connect to a server?
<dretch_> <Sir_Toodles> i only have the one atm although im looking into getting another, and i have not yet tried I happen to be a little busy atm
<dretch_> <Sir_Toodles> I can't find the right resolution size, is there a recommended size to use?
<dretch_> <SirToodles> https://cdn.discordapp.com/attachments/193638031478095872/1018773414221205514/unknown.png
<dretch_> <Sir_Toodles> otherwise it looks like this
<Sweet> if you are not sure about the resolution, you can try 1920x1080
<dretch_> <illwieckz> if you do a fullscreen screenshot we will able to tell your screen resolution 😅
<dretch_> <illwieckz> if you do a fullscreen screenshot we will be able to tell your screen resolution 😅
<dretch_> <Sir_Toodles> the 1920x1080 worked i just couldn't read the graphic options very well
<dretch_> <Sir_Toodles> ive got 4k res
<Sweet> 4k would typically be 3840x2160
```

**Comment by slipher** [comment]
I believe this is the same as https://github.com/Unvanquished/Unvanquished/issues/2243 (fixed by #952).

**Comment by illwieckz** [comment]
Yes, I got this on Windows this week with a 0.54 build. Definitely looks like the issue that is expected to be fixed by #952.
